packages/transaction-controller/src/utils.ts

```diff
@@ -209,26 +209,19 @@ export function getAndFormatTransactionsForNonceTracker(
 }
 
 /**
- * Checks whether a given transaction matches the specified network or chain ID.
- * This function is used to determine if a transaction is relevant to the current network or chain.
+ * Checks whether a given transaction matches the specified chain ID.
+ * This function is used to determine if a transaction is relevant to the current chain.
  *
  * @param transaction - The transaction metadata to check.
  * @param chainId - The chain ID of the current network.
- * @param networkId - The network ID of the current network.
- * @returns A boolean value indicating whether the transaction matches the current network or chain ID.
+ * @returns A boolean value indicating whether the transaction matches the current chain ID.
  */
-export function transactionMatchesNetwork(
+export function transactionMatchesChainId(
```

Comment:
Minor, but rather than making the name so explicit, it could probably stay as `transactionMatchesNetwork` or `transactionMatchesChain` at least.